lib/item_repository.rb

```diff
@@ -85,4 +85,8 @@ def delete(id)
     @items.delete(find_by_id(id))
   end
 
+  def item_count
+    all
+  end 
```

Comment:
we don't need this method at all. we can remove this and simply call `merchants.all` from our sales_engine